content/post/kubedb-v2024.01.31/index.md

```diff
@@ -0,0 +1,470 @@
+
+```yaml
+apiVersion: kafka.kubedb.com/v1alpha1
+kind: ConnectCluster
+metadata:
+  name: connect-cluster
+  namespace: demo
+spec:
+  connectorPlugins:
+  - gcs-0.13.0
+  - mongodb-1.11.0
+  - mysql-2.4.2.final
+  - postgres-2.4.2.final
+  - s3-2.15.0
+  kafkaRef:
+    name: kafka-prod
+    namespace: demo
+  replicas: 2
+  enableSSL: true
+  tls:
+    issuerRef:
+      apiGroup: cert-manager.io
+      kind: Issuer
+      name: connect-ca-issuer
+  terminationPolicy: WipeOut
+  version: 3.6.0
+```
+
+Once ConnectCluster is ready, you can connect any of supported connector via a `Connector` custom resource. In order to Connect a MongoDB database to the `ConnectCluster` apply the following yaml. Provide a kubernetes secret reference in `spec.configSecret` field which should be containing all the necessary connection information to the sink/source client.
+
+```yaml
+apiVersion: kafka.kubedb.com/v1alpha1
+kind: Connector
+metadata:
+  name: mongo-source
+  namespace: demo
+spec:
+  configSecret:
+    name: mongo-source
+  connectClusterRef:
+    name: connect-cluster
+    namespace: demo
+  terminationPolicy: WipeOut
+```
+
+#### supported versions
+`3.3.2`, `3.4.1`, `3.5.1`, `3.6.0`
+
+# Zookeeper
+
+With This release, KubeDB is bringing support for **Apache ZooKeeper**, a pivotal open-source distributed coordination service for managing and synchronizing distributed systems. It offers essential primitives like configuration management, distributed locking, leader election, and naming services. Utilizing a quorum-based approach, ZooKeeper ensures high availability and fault tolerance. Widely applied in systems like Apache Hadoop, Solr, Druid and Kafka, ZooKeeper streamlines development by providing a consistent foundation for coordination tasks. This release of Zookeeper is bringing support for Provisioning and authentication via kubernetes secret.
+
+Apply the following YAML to deploy a ZooKeeper cluster with 3 replicas. 
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: ZooKeeper
+metadata:
+  name: zk-cluster
+  namespace: demo
+spec:
+  version: "3.9.1"
+  replicas: 3
+  storage:
+    resources:
+      requests:
+        storage: "100Mi"
+    storageClassName: "standard"
+    accessModes:
+    - ReadWriteOnce
+  terminationPolicy: "Halt"
+```
+
+#### supported versions
+`3.7.2`, `3.8.3` and `3.9.1`
+
+# Solr
+
+This release includes support for **Solr**, an open-source enterprise-search platform, written in Java. Its major features include full-text search, hit highlighting, faceted search, real-time indexing, dynamic clustering, database integration, NoSQL features and rich document handling. This release of Solr is coming with support for Provisioning SolrCloud cluster, Embedded Admin UI, Custom Configuration via kubernetes secret. You can deploy Solr either in combined mode or the production preferred topology mode.
+
+Here's a sample yaml for provisioning Solr using KubeDB.
```

Comment:
A ZooKeeper instance zk-cluster should be created in the same namespace before deploying it. @pritamdas99